modules/core/shared/src/main/scala/lucuma/core/enum/package.scala

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core
+
+import java.time.Month
+import java.time.Month._
+import cats.implicits._
+import lucuma.core.optics.syntax.ToPrismOps
+
+// The members of this package are generated from database tables, which are the source of truth.
+// See project/gen2.scala for details. Associations with other model types, as needed, are provided
```

Comment:
We can delete the first two sentences here since they no longer apply.

Reply:
:+1:

Reply:
Hmm... Actually we could move the extra methods to the enums themselves then. I'll do that.

Reply:
Oh, right, that makes more sense now.